icu4c/source/test/intltest/rbbitst.cpp

```diff
@@ -2594,17 +2594,6 @@ void RBBITest::TestWordBreaks()
         UnicodeString ustr = CharsToUnicodeString(strlist[loop]);
         // RBBICharMonkey monkey;
         RBBIWordMonkey monkey;
-        if (monkey.dictionarySet().containsSome(ustr)) {
-            // Some of these twenty-year-old random examples depend on the
-            // monkey tests not looking across dictionary/non-dictionary
-            // boundaries for context when applying the rules.
-            // The monkeys are not designed to work with dictionary characters,
-            // so this behaviour is out of scope for testing against the
-            // monkeys.
-            logKnownIssue("ICU-22984");
```

Comment:
Considering my comment, I think the correct resolution is just to dump any of the test cases for which the condition on line 2597 is true.

It is quite late so I am not going to add a trace and compile and run this test now to figure out which ones those are, but I will look at it tomorrow if you don’t beat me to it.

Reply:
Robin fixed this in #3400